repos/spack_repo/builtin/packages/slate/package.py

```diff
@@ -110,6 +114,8 @@ class Slate(CMakePackage, CudaPackage, ROCmPackage):
     depends_on("comgr", when="+rocm")
     depends_on("rocblas", when="+rocm")
     depends_on("rocsolver", when="+rocm")
+    depends_on("cuda@11:", when="@2025.05.28 +cuda") # for c++17 support
```

Comment:
If style doesn't post the fix it said it made, the following needs to be committed:

```suggestion
    depends_on("cuda@11:", when="@2025.05.28 +cuda")  # for c++17 support
```